### ISSUE TITLE
Cost of Good is still not being accurate

### ISSUE THREAD

**Issue by Trevorshaw7** [Repair-DeskPOS/RepairDesk-BUGS-IMPROVEMENTS#630]
When we are changing cost of Goods it does not actually change.  this is causing our reporting to be off.
<img width="1428" alt="screen shot 2019-01-28 at 11 49 09 am" src="https://user-images.githubusercontent.com/43588617/51858857-f4713280-22f2-11e9-8c72-54523df301ca.png">
<img width="1425" alt="screen shot 2019-01-28 at 11 48 59 am" src="https://user-images.githubusercontent.com/43588617/51858872-fb984080-22f2-11e9-8aaf-1647e6ddca11.png">

**Comment by Trevorshaw7** [comment]
<img width="1178" alt="screen shot 2019-01-28 at 11 55 59 am" src="https://user-images.githubusercontent.com/43588617/51859174-c50ef580-22f3-11e9-925b-ba5ea8843159.png">
<img width="1422" alt="screen shot 2019-01-28 at 11 56 14 am" src="https://user-images.githubusercontent.com/43588617/51859197-d0fab780-22f3-11e9-909b-cffc0f331e45.png">

**Comment by Trevorshaw7** [comment]
This shows the cost price update in the admin history but not in the cost of the part
<img width="1423" alt="screen shot 2019-01-28 at 12 02 24 pm" src="https://user-images.githubusercontent.com/43588617/51859562-a826f200-22f4-11e9-989d-4d3aa7aaf3cb.png">

**Comment by Uzair-RepairDesk** [comment]
Hey @Trevorshaw7 
We have tried to reproduce this issue at your end with all the possible scenario’s but couldn’t find the exact scenario that made this issue. 
It certainly should update the COGS if adjustment is made from Goods receiving note or adjustment. 
Fayyaz has placed a specific parameter is the database that will allow us to receive an error log email and we can figure out the reason if this happens again. 
**However, we have updated the COGS records that were not logged correctly.**
